app/controllers/goals_controller.rb

```diff
@@ -22,7 +22,7 @@ def create
     @goal.charity = Charity.find_by(name: params[:charity_selector])
     @goal.tender = User.find_by(username: params[:goal][:tender])
     if @goal.save
-     redirect goal_path ##TAKE ME TO PROFILE PAGE
+      redirect goal_path(@goal)
```

Comment:
Should this be redirect_to rather than just redirect? Think redirect is Sinatra....